src/server/routes/api/v1/parking-lot/index.ts

```diff
@@ -0,0 +1,106 @@
+import httpResponse from '../../../../utils/http-response';
+import { UnexistentParkingLotID } from '../../../../error/parking-lot.service';
+import type {
+  FastifyError,
+  FastifyInstance,
+  FastifyRegisterOptions,
+  FastifyReply,
+  FastifyRequest,
+} from 'fastify';
+import validationSchema from './validation-schema';
+
+export default function (
+  fastify: FastifyInstance,
+  _: FastifyRegisterOptions<unknown>,
+  done: (err?: FastifyError) => void,
+): void {
+  fastify.post(
+    '/',
+    { schema: validationSchema.createParkingLotSchema },
+    async (
+      request: FastifyRequest<{
+        Body: {
+          name: string;
+        };
+      }>,
+      reply: FastifyReply,
+    ) => {
+      try {
+        const parkingLot = await fastify.services.parkingLot.create(
+          request.body,
+        );
+
+        return httpResponse.created(reply, parkingLot);
+      } catch (error) {
+        return httpResponse.internalServerError(reply, error);
+      }
+    },
+  );
+
+  fastify.post(
+    '/:id',
+    { schema: validationSchema.updateParkingLotSchema },
+    async (
+      request: FastifyRequest<{
+        Params: {
+          id: string;
+        };
+        Body: {
+          name: string;
+        };
+      }>,
+      reply: FastifyReply,
+    ) => {
+      try {
+        await fastify.services.parkingLot.findById(request.params.id);
```

Comment:
The resource returned by this function call is not is not being used

Reply:
This is being done twice, once here in L55 of this file and the second time here, on L32 of the service:
https://github.com/Davejs136/thruway-parking/blob/2a00638c8555eaf50d111b0336bf4f05031e86fd/src/server/service/parking-lot.ts#L32

I suggest removing this line from here